collections/SubredditLists.js

```diff
@@ -44,6 +44,19 @@ export class SubscribedSubreddits extends SubredditList {
 
 export class ModeratingSubreddits extends SubredditList {
   static sort = 'mine/moderator';
+
+  static async fetch(apiOptions, options={}, all=true) {
+    const { get } = SubredditEndpoint;
+    console.log({...this.baseOptions(apiOptions), ...options });
+    const allMergedSubreddits = await fetchAll(get, apiOptions, {
+      ...this.baseOptions({
+        ...apiOptions,
+        'view': 'mod'
```

Comment:
`baseOptions` won't use `view` anymore, should this be `sort` or be one-level out in nesting (this way you could merge it onto the result of `baseOptions`)?. If it's `sort` you could change the static property